tests/Rector/v8/v0/Core/Html/Fixture/refactor_removed_marker_methods_from_html_parser.php.inc

```diff
@@ -50,14 +49,17 @@ final class HtmlParserMarkerRendererMethods
         $markerArray = [];
         $subpartArray = [];
         $htmlparser = GeneralUtility::makeInstance(HtmlParser::class);
-        $template = GeneralUtility::makeInstance(MarkerBasedTemplateService::class)->getSubpart($this->config['templateFile'], '###TEMPLATE###');
-        $html = GeneralUtility::makeInstance(MarkerBasedTemplateService::class)->substituteSubpart($html, '###ADDITONAL_KEYWORD###', '');
-        $html2 = GeneralUtility::makeInstance(MarkerBasedTemplateService::class)->substituteSubpartArray($html2, []);
-
-        $content = GeneralUtility::makeInstance(MarkerBasedTemplateService::class)->substituteMarker($content, $marker, $markContent);
-        $content .= GeneralUtility::makeInstance(MarkerBasedTemplateService::class)->substituteMarkerArray($content, $markContentArray, $wrap, $uppercase, $deleteUnused);
-        $content .= GeneralUtility::makeInstance(MarkerBasedTemplateService::class)->substituteMarkerAndSubpartArrayRecursive($content, $markersAndSubparts, $wrap, $uppercase, $deleteUnused);
-        $content = $htmlparser->HTMLcleaner($content);
+        $template = $htmlparser->getSubpart($this->config['templateFile'], '###TEMPLATE###');
+        $html = $htmlparser->substituteSubpart($html, '###ADDITONAL_KEYWORD###', '');
+        $html2 = $htmlparser->substituteSubpartArray($html2, []);
+
+        $html3 = $htmlparser->processTag($value, $conf, $endTag, $protected = 0);
+        $html4 = $htmlparser->processContent($value, $dir, $conf);
+
+        $content = $htmlparser->substituteMarker($content, $marker, $markContent);
+        $content .= $htmlparser->substituteMarkerArray($content, $markContentArray, $wrap, $uppercase, $deleteUnused);
+        $content .= $htmlparser->substituteMarkerAndSubpartArrayRecursive($content, $markersAndSubparts, $wrap, $uppercase, $deleteUnused);
+        $content = $htmlparser->XHTML_clean($content);
```

Comment:
Not sure, but it seems this test case was broken and now its fixed.
Could you confirm?

Reply:
@TomasVotruba This seems wrong to me. I am gonna fix it.